cmd/root.go

```diff
@@ -116,6 +117,8 @@ func AddCommands(cmd *cobra.Command) {
 		config.TIME.DefaultTimeoutLatency, "Default timeout to ntp query")
 	cmd.PersistentFlags().DurationVar(&config.TIME.RefreshNtpInterval, "refresh-ntp-interval",
 		config.TIME.RefreshNtpInterval, "Refresh intervals to ntp")
+	cmd.PersistentFlags().StringSliceVar(&config.TIME.NTPServers,
+		"ntp-servers", config.TIME.NTPServers, "Load NTP servers URLs. The total NTP server URLs should be greater than number of NTP queries")
```

Comment:
I think this message could be a little clearer, how's this?

```suggestion
		"ntp-servers", config.TIME.NTPServers, "A list of NTP servers to query (e.g., 'time.google.com'). Overrides the list in config. Must contain more servers than the number of ntp-queries.")
```